docs/examples/opentracing/main.py

```diff
@@ -3,7 +3,7 @@
 from rediscache import RedisCache
 
 from opentelemetry import trace
-from opentelemetry.exporter.jaeger.thrift import JaegerExporter
+from opentelemetry.exporter.otlp.proto.grpc.trace_exporter import OTLPSpanExporter
```

Comment:
I think something in your dev environment might keep changing this back to one line. The [last lint](https://github.com/open-telemetry/opentelemetry-python/actions/runs/9812174720/job/27100219839?pr=4023) is  still asking for this:

```suggestion
from opentelemetry.exporter.otlp.proto.grpc.trace_exporter import (
    OTLPSpanExporter,
)
```

Reply:
I've committed this change this time. Let's see how it goes 🤞

Reply:
Lint passed yay